drift/core/psestimation.py

```diff
@@ -592,31 +597,28 @@ def fisher_bias(self):
 
     # ====== Estimate the q-parameters from data ========
 
-    def q_estimator(self, mi, vec1, vec2=None, noise=False):
-        """Estimate the q-parameters from given data (see paper).
-
+    def project_vector_kl_to_sky(self, mi, vec1, vec2=None):
```

Comment:
Why does this take two `vec` arguments? It seems like a strange API.

Reply:
changed that to have only vec1.